src/components/Classes/Classes.tsx

```diff
@@ -87,6 +98,7 @@ export const Classes = (p: ClassesProps) => {
 					},
 				],
 			});
+			showRequestNotification();
```

Comment:
Nope. Co jak `addRequest` zrzuci błąd?  Dodać to do callbacka `onSuccess()` w `.mutation.ts`

Jak jest i notka o sukcesie to o błędzie też fajnie dodać na `onError()`